### PR TITLE
reset timer

### PR DESCRIPTION
**Pull request by qq827435393**
i meet a problem： when i transfer a big file, timer is always expired. so how can i resolve it? this way is ok.

### CODE REVIEW

**Comment by qq827435393** [comment]
@googlebot I fixed it.

File: expect.go [hunk 1]
```diff
@@ -799,9 +799,14 @@ func (e *GExpect) ExpectSwitchCase(cs []Caser, timeout time.Duration) (string, [
 			}
 		case <-e.rcv:
 			// Data to fetch.
-			if _, err := io.Copy(&tbuf, e); err != nil {
+			nr, err := io.Copy(&tbuf, e)
+            if err != nil {
 				return tbuf.String(), nil, -1, fmt.Errorf("io.Copy failed: %v", err)
 			}
+            // now should reset timer??
```
Comment (skalle) [inline expect.go:806]: Could you expand the comment a bit. 

What does 0 signify and > 0 signify , why do we only need to reset the timer when it's larger than 0 ..

Comment (qq827435393) [reply expect.go:806]: timer should means  that, there was no new out put data in the past specific secdons. So timer shoud be reset when new out put is available.

Comment (skalle) [reply expect.go:806]: Yeah , just put that in the comment and I'll LGTM.

Comment (qq827435393) [reply expect.go]: PTAL ,（:

File: expect.go [hunk 2]
```diff
@@ -800,13 +800,13 @@ func (e *GExpect) ExpectSwitchCase(cs []Caser, timeout time.Duration) (string, [
...
 			nr, err := io.Copy(&tbuf, e)
-            if err != nil {
+		    	if err != nil {
 				return tbuf.String(), nil, -1, fmt.Errorf("io.Copy failed: %v", err)
 			}
-            // now should reset timer??
-            if nr > 0 {
-                timer = time.NewTimer(timeout)
-            }
+			// timer shoud be reset when new out put is available.
```
Comment (skalle) [inline expect.go:806]: out put -> output

Comment (qq827435393) [reply expect.go]: done

**Comment by skalle** [comment]
Thx a lot for contributing to GoExpect!.

**Pull request merged**